docs/resources/ui/pages/intro-pages.md

```diff
@@ -22,8 +22,7 @@ In FlutterFlow, you can craft a page tailored to your expertise, design preferen
 use case needs. Whether you're starting from scratch, using a template, or leveraging AI tools,
 there are several pathways to achieve the desired functionality and aesthetic of your desired Page.
 
-FlutterFlow allows you to easily create new pages using the Page Selector
-button, which is available in the **Widget Tree menu (Tab+2)**. This will help you
+FlutterFlow allows you to easily create new pages using the "Add Page, Component, or Flow" button, which is available in the **Page Selector menu (Tab+2)**. This will help you
```

Comment:
Thanks for the button name correction! However, Page Selector menu or Widget Tree menu, both are correct, depending on the view you are in. If you have separated widget tree from page view, you may have widget tree on Tab+3 and page selector on Tab+2 but if you combine views, then its just widget tree. Maybe we just say "from Pages view"

Reply:
```suggestion
FlutterFlow allows you to easily create new pages using the "Add Page, Component, or Flow" button, which is available from the Pages view. This will help you
```